### ISSUE TITLE
Not opening/working

### ISSUE THREAD

**Issue by ashgithub2100** [BigBoiCJ/SteamAutoCracker#38]
i downloaded everything it said and tried doing the module request but it doesnt work i did download python too still not opening what should i do?

**Comment by BigBoiCJ** [comment]
Please run the compiled version if you don't manage to run the python script, as I described in #37 

https://github.com/BigBoiCJ/SteamAutoCracker/releases

https://github.com/BigBoiCJ/SteamAutoCracker/issues/37#issuecomment-1828445941

**Comment by ashgithub2100** [comment]
> Please run the compiled version if you don't manage to run the python script, as I described in #37
> 
> https://github.com/BigBoiCJ/SteamAutoCracker/releases
> 
> [#37 (comment)](https://github.com/BigBoiCJ/SteamAutoCracker/issues/37#issuecomment-1828445941)

![Desktop Screenshot 2023 12 01 - 13 55 05 87](https://github.com/BigBoiCJ/SteamAutoCracker/assets/151901423/dd2b4726-e439-497e-a935-b4616e2fd0e7)
![Desktop Screenshot 2023 12 01 - 13 55 05 87](https://github.com/BigBoiCJ/SteamAutoCracker/assets/151901423/6759a39e-5d8e-4533-9c3b-1b6ef6d6fd31)

**Comment by ashgithub2100** [comment]
> Please run the compiled version if you don't manage to run the python script, as I described in #37
> 
> https://github.com/BigBoiCJ/SteamAutoCracker/releases
> 
> [#37 (comment)](https://github.com/BigBoiCJ/SteamAutoCracker/issues/37#issuecomment-1828445941)

i am getting this new type of issue after resolving the opening stuf and here it is pls recommend me what to do:
[1/4] Retrieving game informations from Steam...
- Game found! Name: F1® 23 - AppID: 2108330

[2/4] Retrieving DLCs...
- Found DLC 1/2: F1® 23: 5,000 PitCoin (2502610)
- Found DLC 2/2: F1® 23 Champions Upgrade (2321671)

[3/4] Searching the game folder...

[!!!] A Python error occurred! Writing the error to the error.log file.
---
Traceback (most recent call last):
  File "C:\Users\ash76\OneDrive\Desktop\SteamAutoCracker-1.2.10\steam_auto_cracker.py", line 390, in <module>
    gameDir = FindGameDirectory(gameNameBuffer, platform)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ash76\OneDrive\Desktop\SteamAutoCracker-1.2.10\steam_auto_cracker.py", line 115, in FindGameDirectory
    for folder in os.listdir(config["Locations"][configName]):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: '"C:/Users/ash76/OneDrive/Desktop/f123/F1_23.exe"/'
---

**Comment by BigBoiCJ** [comment]
The error seems to be due to bad configuration on the initial startup.

I advise deleting the `config.ini` file and re-configuring SAC. Please **carefully read the instructions**

You can also manually verify the content of the config file to see if it matches the requirements: https://github.com/BigBoiCJ/SteamAutoCracker/issues/22#issuecomment-1605396995

There are many related issues on this subject #13 https://github.com/BigBoiCJ/SteamAutoCracker/issues/6#issuecomment-1229535796

**Comment by ashgithub2100** [comment]
[3/4] Searching the game folder...
Couldn't automatically find the folder name of the game
Please write the folder name (in SteamApps/common/ or in your pirated games folder)

**Comment by ashgithub2100** [comment]
Couldn't run Steamless on F1_23.exe, it is probably not under DRM.

**Comment by BigBoiCJ** [comment]
F1 2023 uses Denuvo and can't be cracked by SAC. Please check the cs.rin.ru thread.

**Comment by ashgithub2100** [comment]
i have heard denuvo cant be cracked that easily so is there any way possible to crack it? according to your knowledge ????

**Comment by BigBoiCJ** [comment]
@ashgithub2100 nope, you'll have to wait for someone more experienced to do it. Most people have stopped and there's only really one person that's still doing it

Check the cs.rin.ru thread for the corresponding game to check if a crack was released. https://cs.rin.ru/forum/viewtopic.php?f=10&t=118265

Currently F1 2023 hasn't been cracked. Either buy the game or don't play it. Feel free to abuse regional pricing or use key-selling websites, here's a pretty good comparator: https://www.dlcompare.com/

**Comment by ashgithub2100** [comment]
thanks for advicing :)

**Comment by BigBoiCJ** [comment]
I don't know. I have very very little experience.

I only (and vaguely) know the Steam API DRM and the SteamStub DRM

For simple Steam API DRM bypasses, check Steam emulators (cs.rin.ru has a list of them, you could check Goldberg's one which is open source). That's what the `steam_api.dll` file does. SAC uses the ALI213 one.

For SteamStub, it's basically a Steam DRM bundled into the game's .exe file. It can be relatively easily removed if you know how it works. Most of us just use Steamless (SAC uses Steamless).

---
TL;DR I suck at hacking, retroengineering and all that stuff. I'm not even really experienced in programming, don't even know C# and many mainstream languages.

Sadly I'm not the best source for that kind of answers.
